pkg/upstreamyaml/upstreamyaml.go

```diff
@@ -118,7 +118,7 @@ func Parse(upstreamYamlPath string) (*UpstreamYaml, error) {
 	return upstreamYaml, nil
 }
 
-func Write(upstreamYamlPath string, upstreamYaml UpstreamYaml) error {
+func Write(upstreamYamlPath string, upstreamYaml *UpstreamYaml) error {
```

Comment:
Is this change intended?

Reply:
Yes, it is intended. It isn't a functional change - the only purpose is to avoid having to use the `*` operator to dereference `packageWrapper.UpstreamYaml` where `upstreamyaml.Write()` is called in main.go. `packageWrapper.UpstreamYaml` is of type `*upstreamyaml.UpstreamYaml`.